ci.sh

```diff
@@ -60,7 +60,7 @@ case $HOST_TARGET in
     MIRI_TEST_TARGET=i686-unknown-linux-gnu run_tests
     MIRI_TEST_TARGET=aarch64-apple-darwin run_tests
     MIRI_TEST_TARGET=i686-pc-windows-msvc run_tests
-    MIRI_TEST_TARGET=x86_64-unknown-freebsd run_tests_minimal hello integer vec current_dir data_race env
+    MIRI_TEST_TARGET=x86_64-unknown-freebsd run_tests_minimal hello integer vec current_dir data_race env concurrent-panic
```

Comment:
If this adds support for futexes, shouldn't there be a lot more tests that work now?
concurrent-panic is an *awfully* specific test to check. It's not really about concurrency at all. Which part of the Rust stdlib uses _umtx_op that is called by concurrent-panic? AFAIK it uses an RwLock, is that where it comes from?

Reply:
Answering my own question, it's in condvar.

Specifically I don't think this will give us good test coverage for the new operation, that's why we also want some other tests. At least `sync` should be added, I think. If those all pass, that gives me higher confidence that the implementation is correct.